labs/src/main/scala/org/scalalabs/basic/lab03/OptionExercise.scala

```diff
@@ -5,38 +5,50 @@ import sys._
 package object lab03 {
 
   /**
-   * This map contains sample testdata to clarify this exercise.
-   * It contains key value pairs where:
-   * - the key is a room number
-   * - the value can be:
-   * -- the amount of people in the room (filled: Some("10"), empty: None)
-   * -- the room is not available (Some("locked"))
-   */
+  * This map contains sample testdata to clarify this exercise.
+  * It contains key value pairs where:
+  * - the key is a room number
+  * - the value can be:
+  * -- the amount of people in the room (filled: Some("10"), empty: None)
+  * -- the room is not available (Some("locked"))
+  */
   val sampleRooms = Map(1 -> Some("12"), 2 -> None, 3 -> Some("locked"), 4 -> Some("14"), 5 -> Some("8"), 6 -> Some("locked"))
 }
 
 object OptionExercise01 {
 
   /**
-   * Implement the room state method that should return the state of a room as a String as follows:
-   * - filled: return total people:     E.g: Some("12") is "12"
-   * - locked: return "not available"   E.g. Some("locked") is "not available"
-   * - empty:  return "empty"	        E.g. None is "empty"
-   * - does not exist: 					"not existing"
-   */
+  * Implement the room state method that should return the state of a room as a String as follows:
+  * - filled: return total people:     E.g: Some("12") is "12"
+  * - locked: return "not available"   E.g. Some("locked") is "not available"
+  * - empty:  return "empty"	        E.g. None is "empty"
+  * - does not exist: 					"not existing"
+  */
   def roomState(rooms: Map[Int, Option[String]], room: Int): String = {
-    error("Fix me")
+    val query = rooms.get(room)
```

Comment:
You can make this more Scala-y by replacing the if-elif-else with a pattern match on the query.